docs/configuration/providers/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Configuration Providers
+layout: default
+---
+
+Configuration can typically come from different sources. For exemple, command line or desktop apps usually store the configuration in files the the user home directory; server deployments might use a central configuration database; or during testing, the configuration might be hardcoded in the test.
```

Comment:
typo: `command line or desktop apps usually store the configuration in files in the user home directory`

Reply:
done